### PR TITLE
Delete schedule

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-716

### Change description ###
Annotated the scheduled task and updated the integration test.
Config will be updated in the next PR.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5319479&bid=17409084)

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/SchedulerConfigTest.java [hunk 1]
```diff
@@ -24,7 +24,10 @@
     properties = {
         "scheduling.task.scan.enabled=true",
         "scheduling.task.reupload.enabled=true",
-        "scheduling.task.notifications_to_orchestrator.enabled=true"
+        "scheduling.task.notifications_to_orchestrator.enabled=true",
+        "scheduling.task.notifications_to_orchestrator.enabled=true",
```
Comment (doncem) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/SchedulerConfigTest.java:28]: this is included twice

Comment (apereverzin) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/SchedulerConfigTest.java]: Fixed

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5319479&bid=17409084)

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5319479&bid=17409084)

**Pull request merged**